### PR TITLE
Create from native polls

### PR DESCRIPTION
**Pull request by JosXa**
- Added handler for new native Telegram polls in private chat
- Added ability to create new UPB poll from received native poll (including settings, options, and option-description parsing)
- Added context sensitivity to "init menu", in that it renders slightly differently when the poll was created from predefined information contained in a native poll
- Added error handling for when the users sends a quiz instead of a regular poll
- Changed Poll model to include a `created_from_native` boolean field and added migration script
- Included (only) English translations for new features
- Added /cancel command and new aliases for existing commands

### CODE REVIEW

**Comment by JosXa** [comment]
Basiert auf dem anderen PR, also erst mergen dann den hier anschauen :)

File: pollbot/telegram/callback_handler/mapping.py [hunk 1]
```diff
@@ -11,15 +15,23 @@
     skip_description,
     cancel_creation,
     back_to_creation_init,
-    open_init_anonymization_settings,
+    open_init_anonymization_settings, ask_description,
```
Comment (Nukesor) [inline pollbot/telegram/callback_handler/mapping.py]: Small formatting derp

File: pollbot/telegram/native_poll_handler.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+@run_async
+@message_wrapper(private=True)
+def create_from_native_poll(bot: Bot, update: Update, session: Session, user: User):
+    native_poll: NativePoll = update.message.poll
+
+    if user.current_poll is not None and not user.current_poll.created:
+        update.effective_chat.send_message(
+            # Would you like to create a new poll from {poll.question}?
```
Comment (JosXa) [inline pollbot/telegram/native_poll_handler.py]: - [x] remove line

**Review by JosXa — COMMENTED** [review]
- [x] Fix anonymity parsing (always chooses "visible")

File: pollbot/models/poll.py [hunk 3]
```diff
@@ -75,6 +76,7 @@ class Poll(base):
     # Chat state variables
     expected_input = Column(String)
     in_settings = Column(Boolean, nullable=False, default=False)
+    created_from_native = Column(Boolean, nullable=False, default=False)
```
Comment (Nukesor) [inline pollbot/models/poll.py]:
```suggestion
    created_from_native = Column(Boolean, nullable=False, server_default='False', default=False)
```
Please add it in here as well, so the schema defined in the code is in sync to the database schema.

File: pollbot/telegram/callback_handler/creation.py [hunk 4]
```diff
@@ -53,20 +56,37 @@ def open_init_anonymization_settings(session, context, poll):
...
+        # TODO: what if user changed poll type?
+        create_poll(session, poll, context.user, context.tg_chat, context.query.message)
 
 
 @poll_required
 def show_poll_type_keyboard(session, context, poll):
-    """Show to vote type keyboard."""
+    """Show to keyboard to change poll type."""
+
+    # TODO: Review if the poll isn't already available from arguments..? @Nukesor
```
Comment (Nukesor) [inline pollbot/telegram/callback_handler/creation.py:89]: Jep, it's definitely already there.
That's probably some really old legacy stuff I forgot to remove while refactoring :sweat_smile:

File: pollbot/telegram/callback_handler/creation.py [hunk 5]
```diff
@@ -53,20 +56,37 @@ def open_init_anonymization_settings(session, context, poll):
...
-    )
+    if len(poll.options) == 0:
+        context.user.expected_input = ExpectedInput.options.name
+        session.commit()
+        context.query.message.edit_text(
+            i18n.t("creation.option.first", locale=context.user.locale),
+            reply_markup=get_open_datepicker_keyboard(poll),
+        )
+    else:  # options were already prefilled e.g. by native poll
+        # TODO: what if user changed poll type?
```
Comment (Nukesor) [inline pollbot/telegram/callback_handler/creation.py:81]: After looking around a little bit, I don't see any problems with this.
Options don't have any special state depending on the poll type.

This should be ok.

**Review by Nukesor — CHANGES_REQUESTED** [review]
Nice and clean PR.

Two things I need to do myself more often:
- Proof read function doc strings
- Finally start using type hints in Python

File: pollbot/telegram/callback_handler/creation.py (hunk 5 above)
Comment (JosXa) [reply pollbot/telegram/callback_handler/creation.py:81]: It's actually an artifact of my first attempt where I tried avoiding to learn how to create a db migration, so now it's as simple as checking `poll.created_from_native` 🙈

File: pollbot/telegram/callback_handler/creation.py (hunk 4 above)
Comment (JosXa) [reply pollbot/telegram/callback_handler/creation.py:89]: xD 
will also fix the doc

**Comment by Nukesor** [comment]
LGTM. Merging.

**Pull request merged**